packages/@aws-cdk-testing/framework-integ/test/aws-elasticloadbalancingv2/test/integ.nlb.ts

```diff
@@ -11,28 +12,41 @@ const vpc = new ec2.Vpc(stack, 'VPC', {
   maxAzs: 2,
 });
 
-const sg = new ec2.SecurityGroup(stack, 'SG', { vpc });
-sg.addIngressRule(ec2.Peer.anyIpv4(), ec2.Port.tcp(443), 'allow https access from the world');
+const backend = new elbv2.ApplicationLoadBalancer(stack, 'Backend', {
+  vpc,
+});
+backend.addListener('Listener', {
+  protocol: elbv2.ApplicationProtocol.HTTP,
+  defaultAction: elbv2.ListenerAction.fixedResponse(200, {
+    contentType: 'application/json',
+    messageBody: JSON.stringify({
+      Message: 'I am ALB!',
+    }),
+  }),
+});
```

Comment:
I changed target to ALB from IP to test reachability client -> nlb -> backend. This can test security group settings via http api call.